.github/workflows/ci.yml

```diff
@@ -270,6 +270,7 @@ jobs:
 
       - name: "Build flox-env-builder"
         run: |
+          git clean -xfd
```

Comment:
question: Why are we running `git clean` so often?
Could we do this once in a separate step?